src/components/BannerItemComponent.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="col-12">
+    <div
+      class="flex flex-column md:flex-row py-3"
+      :class="{ 'border-top-1 surface-border': index !== 0 }"
+    >
+      <div class="w-full md:w-5 relative md:pr-2">
+        <Image
+          preview
+          class="w-full"
+          v-if="banner.image"
+          :src="getBannerImageSrc(banner)"
+          :pt:image:alt="banner.name"
+          :pt="{
+            image: 'w-full'
+          }"
+        />
+        <div v-else class="px-3 py-5 text-xl surface-hover text-center">
+          {{ $t('banner.noBannersFound') }}
+        </div>
+        <!--
+                    Green when active and banner image is present
+                    Yellow when active but banner image is not present
+                    Red when not active
+                -->
+        <Tag
+          :value="banner.active ? 'Active' : 'Not active'"
+          :severity="banner.active ? (banner.image ? 'success' : 'warning') : 'danger'"
+          class="absolute"
+          style="left: 4px; top: 4px"
+        />
+      </div>
+      <div class="flex flex-row justify-content-between w-full md:w-7">
+        <div class="flex flex-column pr-3">
+          <span class="text-xl">{{ banner.name }}</span
+          ><br />
+          <span class="font-italic">
```

Comment:
These are not properly aligned around the horizontal axis. Could be fixed with some nice flexbox.

Reply:
Done!